OSOE-363: Save not just the HTML page source but also all the referenced files on a test failure in Lombiq.UITestingToolbox

Piedone opened the pull request:
[OSOE-363](https://lombiq.atlassian.net/browse/OSOE-363)

Piedone committed:
Updating Selenium.Axe to latest in Lombiq.UITestingToolbox

Piedone committed:
Updating Atata to latest in Lombiq.UITestingToolbox

Piedone committed:
Now saving an MHTML snapshot of the current page on failure in Lombiq.UITestingToolbox

Piedone committed:
Fixing Spelling failing

Piedone committed:
Special branch references for the Spelling workflow

Piedone committed:
Fixing path

Piedone committed:
And now using an actual configuration

Piedone commented on .github/workflows/build-and-test.yml:
```diff
@@ -41,3 +41,5 @@ jobs:
     name: Spelling
     uses: Lombiq/GitHub-Actions/.github/workflows/spelling.yml@dev
+    with:
+      spell-check-this: Lombiq/GitHub-Actions@issue/OSOE-363
```
While this is not needed when we use the default `@dev`, since we frequently adjust the dictionaries in OSOCE, let's leave it here for easy config changes.

dministro replied:
I feel this will cause confusion, but we will see it.

Piedone committed:
Docs

Piedone committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-363

# Conflicts:
#	tools/Lombiq.GitHub.Actions

Piedone committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-363

# Conflicts:
#	tools/Lombiq.GitHub.Actions

dministro committed:
Targeting dev in workflow
Updating submodules:
 - Lombiq.UITestingToolbox: fast-forward
 - Lombiq.GitHub.Actions: fast-forward
